com.jetbrains.IntelliJ-IDEA-Ultimate.yaml

```diff
@@ -141,6 +141,19 @@ modules:
         path: com.jetbrains.IntelliJ-IDEA-Ultimate.metainfo.xml
       - type: file
         path: idea.properties
+  - name: git-lfs
+    buildsystem: simple
+    build-commands:
+      - PREFIX=${FLATPAK_DEST} ./install.sh
+    sources:
+    - type: archive
+      url: https://github.com/git-lfs/git-lfs/releases/download/v3.3.0/git-lfs-linux-amd64-v3.3.0.tar.gz
+      sha256: 6a4e6bd7d06d5c024bc70c8ee8c9da143ffc37d2646e252a17a6126d30cdebc1
+      x-checker-data:
+          project-id: 11551
```

Comment:
I think the `4` spaces here will make the YAML file invalid, at least for this section.